04_HowTos/QueryClient/How_to_use_the_Data_Lab_QueryClient.ipynb

```diff
@@ -176,7 +200,7 @@
     }
    ],
    "source": [
-    "query = 'select gmag, rmag, imag from smash_dr1.object limit 10'\n",
+    "query = 'SELECT gmag, rmag, imag FROM smash_dr1.object LIMIT 10'\n",
```

Comment:
The query is SQL, but the call uses ADQL (it works b/c our system converts on the backend. Also, the query as-is returns many '99.99' values, which are 'invalid' encodings in SMASH. Suggest to change this cell to:

```
query = 'SELECT gmag, rmag, imag FROM smash_dr1.object WHERE gmag < 99 LIMIT 10'
response = qc.query(sql = query, fmt = 'csv')
print (response)
```